### PR TITLE
Implementation of KHR_parallel_shader_compile ext

### PR DESCRIPTION
**Pull request by Maksims**
I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/graphics/shader.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ Object.assign(pc, function () {
      * @param {String} definition.vshader Vertex shader source (GLSL code).
      * @param {String} definition.fshader Fragment shader source (GLSL code).
      * @param {Boolean} definition.useTransformFeedback Specifies that this shader outputs post-VS data to a buffer
-     * @param {Object} precache Triggers imediate link.
```
Comment (willeastcott) [inline src/graphics/shader.js]: imediate -> immediate

Comment (Maksims) [reply src/graphics/shader.js]: This line is removed in this PR, as there is no such argument.

**Comment by willeastcott** [comment]
Hey @Maksims - why did you decide to close this? I didn't feel it was quite ready but what was your reason?

**Comment by Maksims** [comment]
Must have missclicked.
What you believe is required to make it ready?

**Pull request closed without merging**